README.md

```diff
@@ -170,6 +119,12 @@ this is mapped to memory. This is intentional, so make sure you have enough
 RAM or you can experience some swapping. Alternatively, change the temp
 directory in the scripts.
 
+It is possible to modify SYSLINUX kernel command line by changing
+livecd-creator code in /usr/lib/python2.7/site-packages/imgcreate/live.py
```

Comment:
I have a TODO item to expand this. Existing image should be modifiable, don't know how yet. Will rewrite this section.